DEPLOYMENT.md

```diff
@@ -13,37 +15,42 @@ javascripting | 2.6.1 | [2.6.1](https://github.com/workshopper/javascripting) |
 
 ## Requirements
 
-Please be kind to prepare the following prerequisites before asking or publishing your package to npm:
+Please prepare the following prerequisites for package deployment:
 
-- New version of the package should follow semantic versioning
-- Ensure that the github repository of the workshopper is correctly setup and working with [Travis CI](http://travis-ci.org/).
-- Master branch of workshopper github repository is "buildable", i.e. not broken (Tests are passing, and the lint and other code standards changes are acceptable)
-- Comprehensible change log is provided on the workshopper github master branch in a CHANGELOG.md file. This could be made manually while making a new release on the repository, or prepared automatically from a script (read further below for suggestions)
+- New version of the package should follow [semantic versioning](http://semver.org/);
```

Comment:
Agree, this must be a bad habit of terminating a line of code. Changed to full stops.